pkg/Directory.Build.props

```diff
@@ -33,6 +33,8 @@
     <Content Include="$(PackageAssetsPath)$(PackageIdFolderName)\lib\**\*" Pack="true" PackagePath="lib" />
     <Content Include="$(PackageAssetsPath)$(PackageIdFolderName)\analyzers\**\*" Pack="true" PackagePath="analyzers" />
     <Content Include="$(PackageAssetsPath)$(PackageIdFolderName)\runtimes\**\*" Pack="true" PackagePath="runtimes" />
+    <Content Include="$(RepoRoot)\THIRD-PARTY-NOTICES.TXT" Pack="true" PackagePath="\" />
```

Comment:
Actually, there are a few packages we don't want these in.  For example, TensorFlow.Redist and Mkl.Redist.

To make that happen you can do the following.  In this file do this:

```XML
<ItemGroup Condition="'$(IncludeMLNetNotices)' != 'false'">
  <Content Include="$(RepoRoot)\THIRD-PARTY-NOTICES.TXT" Pack="true" PackagePath="\" />
  <Content Include="$(RepoRoot)\LICENSE" Pack="true" PackagePath="\" />
</ItemGroup>
```

And then in the TF.Redist and MKL.Redist (and any other nuget that we don't want these in), you put:

```XML
<PropertyGroup>
  <IncludeMLNetNotices>false</IncludeMLNetNotices>
</PropertyGroup>
```